sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -53,21 +56,23 @@ internal class OperationInternal<T> : OperationInternalBase
     {
         private readonly IOperation<T> _operation;
         private readonly AsyncLockWithValue<OperationState<T>> _stateLock;
-        private Response _rawResponse;
+        private Response? _rawResponse;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OperationInternal"/> class in a final successful state.
         /// </summary>
         /// <param name="rawResponse">The final value of <see cref="OperationInternalBase.RawResponse"/>.</param>
         /// <param name="value">The final result of the long-running operation.</param>
-        public static OperationInternal<T> Succeeded(Response rawResponse, T value) => new(OperationState<T>.Success(rawResponse, value));
+        /// <param name="requestMethod">rehydration token</param>
+        public static OperationInternal<T> Succeeded(Response rawResponse, T value, RequestMethod? requestMethod = null) => new(OperationState<T>.Success(rawResponse, value), requestMethod);
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OperationInternal"/> class in a final failed state.
         /// </summary>
         /// <param name="rawResponse">The final value of <see cref="OperationInternalBase.RawResponse"/>.</param>
         /// <param name="operationFailedException">The exception that will be thrown by <c>UpdateStatusAsync</c>.</param>
-        public static OperationInternal<T> Failed(Response rawResponse, RequestFailedException operationFailedException) => new(OperationState<T>.Failure(rawResponse, operationFailedException));
+        /// <param name="requestMethod">rehydration token</param>
```

Comment:
The `requestMethod` comment seems wrong.   Please correct and format as a sentence to match existing docs.

Reply:
updated